health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/ChooseActivity.js

```
@@ -52,6 +53,16 @@ const ChooseActivity = () => {
         <React.Fragment>
             <ActivityHomeCard title={t("SELECT_AN_ACTIVITY_TO_CONTINUE")} module={activityCardData}/>
+            <InboxFilterWrapper
+             options={[
+                { code: "M", name: "Male" },
+                { code: "F", name: "Female" },
+                { code: "O", name: "Others" }
+              ]}
+             onApplyFilters={(filterData) => console.log("Applied Filters:", filterData)}
+             optionsKey={"name"}
+             defaultValue={{ code: "M", name: "Male" }}
+            ></InboxFilterWrapper>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)